django/conf/global_settings.py

```diff
@@ -546,6 +546,8 @@ def gettext_noop(s):
 
 AUTH_PASSWORD_VALIDATORS = []
 
+CHECK_PASSWORD_EXECUTOR_MAX_WORKERS = None
```

Comment:
We shouldn't add a global setting for something niche within a contrib app
In general adding settings is quite controversial and needs to be discussed and agreed with the wider community

Reply:
Thanks for checking. If we don't want to control any of the `ThreadPoolExecutor` options, we can also rely on its basic instance or maybe create a ThreadPoolExecutor every `acheck_password` call, depending on the performance.